tests/tests/test_filetransfer.py

```diff
@@ -70,15 +74,52 @@ def upload_file(path, file, devid, authtoken, mode="600", uid="0", gid="0"):
     return requests.put(upload_url, verify=False, headers=authtoken, files=files)
 
 
+def set_limits(mender_device, limits):
+    tmpdir = tempfile.mkdtemp()
+    try:
+        # retrieve the original configuration file
+        output = mender_device.run("cat /etc/mender/mender-connect.conf")
+        config = json.loads(output)
+        # replace mender.conf with an mTLS enabled one
+        config["Limits"] = limits
+        mender_connect_conf = os.path.join(tmpdir, "mender-connect.conf")
+        with open(mender_connect_conf, "w") as fd:
+            json.dump(config, fd)
+        mender_device.run(
+            "cp /etc/mender/mender-connect.conf /etc/mender/mender-connect.conf-backup-`ls /etc/mender/mender-connect.* | wc -l`"
+        )
+        mender_device.put(
+            os.path.basename(mender_connect_conf),
+            local_path=os.path.dirname(mender_connect_conf),
+            remote_path="/etc/mender",
+        )
+    finally:
+        shutil.rmtree(tmpdir)
+    connect_service_name = "mender-connect"
+    mender_device.run("systemctl restart %s" % connect_service_name)
+    # mender_device.run("ln -s /etc/mender/mender-connect.conf /etc/mender/mender-connect-link.conf")
```

Comment:
I believe we can remove this comment, I don't get what it is supposed to do.

Reply:
leftover comment. removed.